frontend/nuxt.config.ts

```diff
@@ -41,4 +44,24 @@ export default defineNuxtConfig({
     ],
     dirs: ['globalimports'],
   },
+  i18n: {
+    // TODO(grady) Set a Base URL once we have it for SEO https://i18n.nuxtjs.org/guide/seo#:~:text=You%20must%20also%20set%20the%20baseUrl%20option%20to%20your%20production%20domain%20in%20order%20to%20make%20alternate%20URLs%20fully%2Dqualified%3A
+    // baseUrl: 'https://my-nuxt-app.com'
```

Comment:
But we do have a base URL! In dev, it's `https://pacta.dev.rmi.siliconally.dev`. Seems like this should be populated from `envs/` like other env-specific config

Reply:
GC fixed